gef.py

```diff
@@ -8851,6 +8857,16 @@ def do_invoke(self, argv):
             err("Unmapped address")
             return
 
+        ref_addr = safe_parse_and_eval(reference)
+        if ref_addr is None:
+            err("Invalid reference")
```

Comment:
Let's add the attempted ref to the error

Reply:
i don't understand. do you mean a more concise error message?

Reply:
I meant more `"Invalid reference '{}'".format(reference)`